client/deposits/list/index.tsx

```diff
@@ -214,7 +214,7 @@ export const DepositsList = (): JSX.Element => {
 
 	const endpointExport = async () => {
 		const userEmail = wcpaySettings.currentUserEmail;
-		const userLocale = wcpaySettings.userLocale.code;
+		const locale = wcSettings.locale.userLocale;
```

Comment:
This is deceptive! 

- Is `wcSettings.locale.userLocale` a ~2-letter code?
- `wcpaySettings.userLocale.code` is a struct with other stuff in it (`code: english_name: ; native_name: ;`) - do we need it in our settings?

Using the store settings seems cleaner to me 🚀

Reply:
> This is deceptive!

(i.e. took me a while to see what was changing)

Reply:
OMG nice, you have removed `wcpaySettings.userLocale` 🫡